kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/MergeQueries.java

```diff
@@ -153,13 +151,13 @@ private void mergeFlush(
           bigQuery.query(QueryJobConfiguration.of(mergeFlushQuery));
           success = true;
         } catch (BigQueryException e) {
-          if (BigQueryErrorResponses.isCouldNotSerializeAccessError(e)) {
+          if (BigQueryErrorResponses.isCouldNotSerializeAccessError(e) || BigQueryErrorResponses.isJobInternalError(e)) {
```

Comment:
Lets use `bigQueryRetry` config to decide the retry count for isJobInternalError